tiledb/api/c_api/array_schema/array_schema_api_internal.h

```diff
@@ -41,27 +41,27 @@
 #include "tiledb/sm/enums/array_type.h"
 #include "tiledb/sm/enums/layout.h"
 
-using namespace tiledb::sm;
-
 /** Handle `struct` for API ArraySchema objects. */
 struct tiledb_array_schema_handle_t
     : public tiledb::api::CAPIHandle<tiledb_array_schema_handle_t> {
   /** Type name */
   static constexpr std::string_view object_type_name{"array_schema"};
 
  private:
-  using array_schema_type = shared_ptr<ArraySchema>;
+  using array_schema_type = shared_ptr<tiledb::sm::ArraySchema>;
```

Comment:
Could you explain why we now need to fully qualify tiledb::sm classes now?

Reply:
Oops, meant to add a comment for this on my PR. It was namespace pollution with `using tiledb::sm` at the top of this file. When I [included this header](https://github.com/TileDB-Inc/TileDB/pull/5291/files#diff-309cba298b83ecf44881628cab047f3e441463ac668c7cec9793e360645f2c30R37) it makes `tiledb::Array` and `tiledb::sm::Array` ambiguous for example.